test/HealthChecks.Elasticsearch.Tests/Functional/ElasticsearchAuthenticationTests.cs

```diff
@@ -25,7 +24,7 @@ public async Task be_healthy_if_elasticsearch_is_using_valid_api_key()
                     .AddElasticsearch(options =>
                     {
                         options.UseServer(connectionString);
-                        options.UseApiKey(new ApiKeyAuthenticationCredentials(_fixture.ApiKey));
+                        options.UseApiKey(_fixture.ApiKey!);
```

Comment:
The test that verifies that the API remains unchanged is failing, please edit [HealthChecks.Elasticsearch.approved.txt](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/test/HealthChecks.Elasticsearch.Tests/HealthChecks.Elasticsearch.approved.txt) to make it pass.